README.rst

```diff
@@ -13,45 +13,36 @@ Beamline Simulation using `sirepo-bluesky`_ and Synchrotron Radiation Workshop (
 
 Purpose
 -------
-
-Use machine learning to simulate beamlines similar to SRW with lower computational costs.
-The goal is to determine whether or not a neural network can construct a beamline simulation given data from sirepo-bluesky.
+Use machine learning techniques to simulate beamlines given data from SRW.
 
 Installation
 ------------
 
 - Install `VirtualBox`_ on your computer.
 - Install `Vagrant`_ using the terminal.
 - Git clone this repository. Then use ``cd deep-beamline-simulation`` to move into the top level directory of this repository.
-- You will see a Vagrantfile containing virtual machine setup information. Start the virtual machine using ``vagrant up`` followed by ``vagrant ssh``.
-- Once the virtual machine is running, finish the Miniconda installation,
-.. code:: bash
+- You will see a Vagrantfile containing virtual machine setup information. Start the virtual machine using ``vagrant up`` followed by ``vagrant ssh``. To reload the virtual machine use ``vagrant reload`` and to update the machine for any changes use ``vagrant provision``.
 
-   bash Miniconda3-latest-Linux-x86_64.sh
-
-- Close and reload the VM. Create a conda environment, 
-.. code:: bash
+- It is recommended to check the status of Mongo DB using ``sudo systemctl status mongod``. If the status is 'dead' use ``sudo systemctl start mongod.service`` to start running Mongo DB.
 
-   conda create -n sirepo_bluesky python=3.8
-   conda activate sirepo_bluesky
+- To view the contents of ``deep-beamline-simulation`` repository use ``cd /vagrant``.
 
-- It is recommended to check the status of Mongo DB using ``sudo systemctl status mongod``. If the status is 'dead' use ``sudo systemctl start mongod.service`` to start running Mongo DB.
+- There will be a conda environment created using the Vagrantfile. Verify this by using ``conda env list``. To activate it use ``conda activate dbs``.
 
-- To view the contents of ``deep-beamline-simulation`` repository use ``cd /vagrant``. Use ``pip install .`` to install all requirements and setup necessary packages. 
+- Use ``pip install .`` to install all requirements and setup necessary packages. 
```

Comment:
Here's some more example output to include (some or all) showing sirepo starting up:
```
vagrant@vagrant:~$ conda env list
# conda environments:
#
base                  *  /home/vagrant/miniconda3
dbs                      /home/vagrant/miniconda3/envs/dbs

vagrant@vagrant:~$ conda activate dbs
(dbs) vagrant@vagrant:~$ cd /vagrant/
(dbs) vagrant@vagrant:/vagrant$ pip install .

...

Successfully built deep-beamline-simulation
Installing collected packages: deep-beamline-simulation, ...

...

Successfully installed deep-beamline-simulation-0.post246.dev0+g97d4ced ...

(dbs) vagrant@vagrant:/vagrant$ bash scripts/start_sirepo.sh -it
Creating Directory /home/vagrant/tmp/data/2021/12/20
beta: Pulling from radiasoft/sirepo
4ec705285520: Already exists
d08afb6fdff6: Already exists
656017a811bb: Already exists
8451f4d1b167: Already exists
acdf072d0fee: Already exists
a2a3dc9db0cb: Pull complete
000d8e6ab74f: Pull complete
Digest: sha256:fba5aea955cda2807daf66443c010d12a22df237be18a39269908c1e4f3e9247
Status: Downloaded newer image for radiasoft/sirepo:beta
docker.io/radiasoft/sirepo:beta
REPOSITORY         TAG       IMAGE ID       CREATED       SIZE
radiasoft/sirepo   beta      5becae748c04   5 days ago    5.76GB
radiasoft/sirepo   <none>    8117306ff3a6   3 weeks ago   5.76GB
radiasoft/sirepo   <none>    9b56b3e3a7ff   5 weeks ago   5.76GB
Command to run:

docker run -it --init --rm --name sirepo        -e SIREPO_AUTH_METHODS=bluesky:guest        -e SIREPO_AUTH_BLUESKY_SECRET=bluesky        -e SIREPO_SRDB_ROOT=/sirepo        -e SIREPO_COOKIE_IS_SECURE=false        -p 8000:8000        radiasoft/sirepo:beta bash -l -c "mkdir -v -p /sirepo && sirepo service http"

...

 * Serving Flask app 'sirepo.server' (lazy loading)
 * Environment: development
 * Debug mode: off
 * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://172.17.0.2:8000/ (Press CTRL+C to quit)
 * Restarting with stat

```

In a new terminal:
```
vagrant@vagrant:~$ docker ps -a
CONTAINER ID   IMAGE                   COMMAND                  CREATED         STATUS         PORTS                                       NAMES
f0d01fee65cf   radiasoft/sirepo:beta   "bash -l -c 'mkdir -…"   9 minutes ago   Up 9 minutes   0.0.0.0:8000->8000/tcp, :::8000->8000/tcp   sirepo

vagrant@vagrant:~$ docker stop sirepo
sirepo
```